### PR TITLE
fixed typos & rm unused code

### PR DESCRIPTION
**Pull request by ndrean**
One tap Google Login: removed some forgotten unused code

### CODE REVIEW

File: lib/page_controller.ex [hunk 1]
```diff
@@ -1,5 +1,5 @@
 defmodule AppWeb.PageController do
-  # use Phoenix.Controller
+  use Phoenix.Controller
```
Comment (nelsonic) [inline lib/page_controller.ex:2]: Can't say I understand what's going on here ... why are we adding a controller to this auth _library_? 💭 
This feels more like an _implementation_ of a One-Tap feature than a library that people can include in their project. 

I'm on-board with adding the One-tap functionality ... 
but this appears to be adding **`Phoenix`** ...
Which isn't our intention with these packages. 

Perhaps I'm missing something here. 🤷‍♂️

Comment (ndrean) [reply lib/page_controller.ex:2]: Humm yes indeed, written this way, it is unclear that this controller is just the source (readme) of the example of how to use the module. 

The only needed libraries are in the Mix.

I didn't add a folder "example" at this stage.  Other files such as "index, app, page" are not part of a ready-to-use package but are the source for Readme. Was just unsure of the interest in such a feature.

Comment (nelsonic) [reply lib/page_controller.ex:2]: Definitely keen on having this feature. But it belongs more in the **`demo`** App: 
https://github.com/dwyl/elixir-auth-google-demo which is a full Phoenix App that demonstrates how to use this package. We could create a new page/template that showcases the One Tap/Click. 💭

Comment (ndrean) [reply lib/page_controller.ex:2]: Absolutely, this doesn't deserve a module in hex, not at all. Just 10 LOC. But  10 "rich" lines I believe, at least not straightforward.
<https://elixirforum.com/t/using-joken-to-validate-google-jwts/19728>

I will push a branch with this, with a One tap readme.

Comment (ndrean) [reply lib/page_controller.ex:2]: And the same idea with the Facebook login if you agree, the SSR module, and add the SDK in the demo.

**Pull request closed without merging**